include/yaml-cpp/contrib/anchordict.h

```diff
@@ -22,6 +22,7 @@ namespace YAML {
 template <class T>
 class AnchorDict {
  public:
+  AnchorDict() : m_data{} {}
```

Comment:
Make sure to run clang-format; I think there's an extra space here.

Reply:
`clang-format` didn't seem to change that.